forge.config.js

```diff
@@ -0,0 +1,62 @@
+const fs = require("fs")
+
+const devClientId = "e56ae4a7249b39454ea5"
+const devClientSecret = "c00b3a34ca123c1e0870007aec67e91d5618bc46"
+
+const s = JSON.stringify
+
+const appInfo = s({
+  client_id: process.env.CLASSROOM_DESKTOP_OAUTH_CLIENT_ID || devClientId,
```

Comment:
Generates app-info.json file which has client id and client secret, checks environment variable during build time or defaults to development id and secret